packages/backend/src/index.ts

```diff
@@ -155,16 +157,39 @@ export function initIocContainer(
       accountingService: await deps.use('accountingService')
     })
   })
+
+  container.singleton('ledgerAccountService', async (deps) => {
```

Comment:
I was thinking of registering this singleton conditionally after L185, but it felt wrong to have an optional singleton resolution